data/src/scripts/skill_agility/scripts/gnome_course.rs2

```diff
@@ -6,6 +6,11 @@ mes("You walk carefully across the slippery log...");
 mes("...You make it safely to the other side.");
 ~update_gnome_varp(1);
 
+// Go to the net
+if (loc_find(0_38_53_41_33, gnome_obstacle_net_1) = true) {
```

Comment:
`loc_find` sets the `active_loc` to whatever the first argument is (in this case `0_38_53_41_33`). Since the command returned true, we know the object also exists at that coordinate. Then calling `p_oploc(1)` will tell the player to interact with the `active_loc` and use the first (`1`) interact option